src/Components/Map/Subcomponents/ItemPopupComponents/TextView.tsx

```diff
@@ -53,6 +53,22 @@ export const TextView = ({
 
   if (innerText) replacedText = fixUrls(innerText)
 
+  if (replacedText) {
+    replacedText = replacedText.replace(/(?<!\]?\()(?<!<)https?:\/\/[^\s)]+(?!\))(?!>)/g, (url) => {
+      let shortUrl = url
+
+      if (url.match('^https://')) {
+        shortUrl = url.split('https://')[1]
+      }
+
+      if (url.match('^http://')) {
+        shortUrl = url.split('http://')[1]
+      }
+
+      return `[${shortUrl}](${url})`
+    })
+  }
```

Comment:
```suggestion
  if (replacedText) {
    replacedText = replacedText.replace(/(?<!\]?\()(?<!<)https?:\/\/[^\s)]+(?!\))(?!>)/g, (url) => `[${url.replace(/https?:\/\/w{3}\./gi,'')}](${url})`
    })
  }
```

Note: this also replaces the `www`